flutter/analysis_options.yaml

```diff
@@ -13,8 +13,7 @@ analyzer:
     deprecated_member_use_from_same_package: warning
     # ignore sentry/path on pubspec as we change it on deployment
     invalid_dependency: ignore
-  exclude:
-    - example/**
+    unnecessary_import: ignore
```

Comment:
Did we not fix all the issues? This is not necessary anymore

Reply:
You're probably right. This must be a merge issue.

Reply:
```suggestion
```